src/pages/sharedNotes/SharedNotes.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Notes from '../myNotes/notes';
+import { PageTitle } from './styled';
+
+const SharedNotes = () => {
+    return (
+        <>
+            <PageTitle>THERE IS A LIST OF SHARED NOTES</PageTitle>
```

Comment:
try to avoid capslock text. Write it in a normal way and if you want it in upper register - use CSS for it

Reply:
Changed